content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+requirements, you likely need clusters across multiple zones and regions. You
+can canary configuration changes or new binary releases in a single cluster,
+where the configuration changes only affect a small amount of user traffic.
+Additionally, if a cluster has a problem, you can temporarily route traffic to
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="50%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="50%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
```

Comment:
@smawson Would this wording work?

```
Workload instances can all share a common identity and authentication infrastructure and can authenticate each other. Istio also supports authentication that only uses shared trust. However, workload instances do have a common identity since service account names are unique within a namespace, just like service names.
```

Reply:
I would just drop the part about common authentication infrastructure or shared trust:

Additionally, workload instances all share a common identity as service accounts names are unique within a namespace, just like service names.